README.md

```diff
@@ -37,15 +37,16 @@ TRACTION_BASE_URL="https://traction-tenant-proxy-dev.apps.silver.devops.gov.bc.c
 TRACTION_WALLET_ID="b1d5b628-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
 TRACTION_WALLET_KEY="286e7818-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
 GOOGLE_AUTH_JSON_PATH="path_to_google_oauth_json_key_from_src.json"
+REQUEST_ATTESTATION_JSON_PATH="fixtures/request_attestation.json"
+OFFER_JSON_PATH="fixtures/offer.json"
```

Comment:
I think this should just be a path to the message templates. That way we can add more as needed and we don'n need to update the OCP deployment.

Reply:
done